platform/source/tools/container_outer_executor.py

```diff
@@ -521,25 +513,25 @@ def _pre_run(self):
             return False
 
         # Adding encrypted strings into a set if there is any in cmd_options or entry point
-        logger.info("Start decrypting.")
-        if self._cmd_options is not None or self._entrypoint is not None:
-            entry_word_list = list()
-            if self._cmd_options is not None:
-                entry_word_list.extend(self._cmd_options.split())
-            if self._entrypoint is not None:
-                entry_word_list.extend(self._entrypoint.split())
-            for entry_word in entry_word_list:
-                match_obj = re.match("%%secrets.(.*)%%", entry_word)
-                if match_obj is not None:
-                    token = match_obj.group(1)
-                    if token and token not in self._encrypted_strings:
-                        try:
-                            self._encrypted_strings[token] = axops_client.decrypt(token=token, repo_name=self._repo)
-                        except Exception as e:
-                            logger.exception("Failed to decrypt %s", token)
-                            self._gen_user_command_with_failed_decryption(token, error_messages=str(e))
-                            self._write_executor_sh()
-                            return False
+        # logger.info("Start decrypting.")
```

Comment:
why comment this out? Remove the code if not needed...

Reply:
@pratulw and @edlee2121 would like to keep the secret management v1. Will make another change regarding to this.